vm/src/types/instance_definitions/builtins_instance_def.rs

```diff
@@ -192,25 +194,54 @@ impl BuiltinsInstanceDef {
         }
     }
 
-    pub(crate) fn dynamic() -> BuiltinsInstanceDef {
+    pub(crate) fn dynamic(params: CairoLayoutParams) -> BuiltinsInstanceDef {
+        let pedersen = Some(PedersenInstanceDef {
+            ratio: Some(params.pedersen_ratio),
+        });
+        let range_check = Some(RangeCheckInstanceDef {
+            ratio: Some(params.range_check_ratio),
+        });
+        let ecdsa = Some(EcdsaInstanceDef {
+            ratio: Some(params.ecdsa_ratio),
+        });
+        let bitwise = Some(BitwiseInstanceDef {
+            ratio: Some(params.bitwise_ratio),
+        });
+        let ec_op = Some(EcOpInstanceDef {
+            ratio: Some(params.ec_op_ratio),
+        });
+        let keccak = Some(KeccakInstanceDef {
+            ratio: Some(params.keccak_ratio),
+        });
+        let poseidon = Some(PoseidonInstanceDef {
+            ratio: Some(params.poseidon_ratio),
+        });
+        let range_check96 = Some(RangeCheckInstanceDef {
+            ratio: Some(params.range_check96_ratio),
+        });
+        let add_mod = Some(ModInstanceDef {
```

Comment:
Why aren't we doing this here?
I think we should add it

```
            #[cfg(feature = "mod_builtin")]
            add_mod: Some(...),
            #[cfg(feature = "mod_builtin")]
            mul_mod: Some(...),
            #[cfg(not(feature = "mod_builtin"))]
            add_mod: None,
            #[cfg(not(feature = "mod_builtin"))]
            mul_mod: None,
```

Reply:
Done